webpack/components/extensions/HostDetails/Cards/ErrataOverviewCard.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import {
+  Card,
+  CardHeader,
+  CardTitle,
+  CardBody,
+  Flex,
+  FlexItem,
+  GridItem,
+} from '@patternfly/react-core';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { propsToCamelCase } from 'foremanReact/common/helpers';
+import PropTypes from 'prop-types';
+import { ChartPie } from '@patternfly/react-charts';
+import { ErrataMapper } from '../../../../components/Errata';
+
+function HostInstallableErrata({
+  id, errataCounts,
+}) {
+  const errataTotal = errataCounts.total;
+  const errataSecurity = errataCounts.security;
+  const errataBug = errataCounts.bugfix;
+  const errataEnhance = errataCounts.enhancement;
+  const chartData = [{ x: 'security', y: errataSecurity }, { x: 'bugfix', y: errataBug }, { x: 'enhancement', y: errataEnhance }];
+  return (
+    <GridItem rowSpan={2} md={3} lg={5}>
+      <Card isHoverable>
+        <CardHeader>
+          <CardTitle>{__('Installable Errata')}</CardTitle>
+        </CardHeader>
+        <CardBody>
+          <Flex direction="column">
+            <FlexItem>
+              <a href={urlBuilder(`content_hosts/${id}/errata`, '')}>
+                {errataTotal} errata
+              </a>
+            </FlexItem>
+            <Flex flexWrap={{ lg: 'nowrap' }} direction="row">
```

Comment:
I found that adding `align-items: center;` to the flex container will fix the vertical spacing.  I think giving the PF `<Flex>` the following props will accomplish that:
```suggestion
            <Flex flexWrap={{ lg: 'nowrap' }} direction="row" alignItems={{ default: 'alignItemsCenter' }}>
```

But yeah, the CSS to remember is, on the flex container `align-items: center;` :)